### ISSUE TITLE
maximum rotation angle for receipt auto crop.

### ISSUE THREAD

**Issue by RowanG1** [hoangphuc3117/cropreceiptdemo#5]
Is there a reason why the maximum rotation angle of receipt is only a few degrees, otherwise cropped box is the original image?

**Comment by hoangphuc3117** [comment]
You mean, when you rotate the receipt with large degrees ( x > 45 and < 90). The algorithm will crop the receipt wrongly?

**Comment by RowanG1** [comment]
Yes. It works fine for up to about 15 degree rotation. But for larger angles, there is basically no crop. I would say considering angles 0 to 45 degrees, 0 to 15 is okay, and 15 to 45 not.  0 to 45 is a similar situation as 45 to 90.

**Comment by hoangphuc3117** [comment]
My app is a demo for 2 features.
1/ Detect automatically corners of the receipt: currently maybe my algorithm doesn't work well in many cases. 
2/ Crop the receipts by 4 points: I don't think the problem happens in this step. Because I only use a method of openCV to crop the receipt. 
Thank you for your feedback. I'll review it when I have free time.
